.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
     - $HOME/.composer/cache/files
 
 php:
-  - 7
+  - 7.1
```

Comment:
Are we confident that 7.1 is stable enough? I know we want new features, and I know this is just a generation tool, but what if there's a segfault or something (happened recently with me on 7.1)... just something to consider.